packages/core/src/utils/helper-functions.js

```diff
@@ -21,3 +23,8 @@ export const debounceEvent = (callback, wait) => {
 		timeout = setTimeout(() => callback.apply(this, args), wait);
 	};
 };
+
+export const isEscapeEvent = event => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS3524)